test/integration/targets/connection_posix/test.sh

```diff
@@ -16,3 +16,14 @@ INVENTORY="../connection_${group}/test_connection.inventory" ./test.sh \
     -e local_tmp=/tmp/ansible-local \
     -e remote_tmp=/tmp/ansible-remote \
     "$@"
+
+DOCKER_VERSION=$(docker version -f '{{ .Client.Version }}')
```

Comment:
It fails with the following error on every build:
```
02:50 ++ docker version -f '{{ .Client.Version }}'
02:50 ./runme.sh: line 20: docker: command not found
02:50 + DOCKER_VERSION=
```
@mattclay isn't `docker` binary available on some build (for example the ones which aren't containers (for example `rhel`) ?

Reply:
It might be done in `test_workdir.yml` playbook using:
```
roles:
  - setup_docker
post_tasks:  # use 'post_tasks' instead of 'tasks' in order to execute the tasks below after the whole 'setup_docker' role
  - name: run pwd
    ...
```

Reply:
Couldn't you also use `aliases` to add `setup_docker` as a dependency?

Reply:
Aren't dependencies defined in `meta/main.yml` ? but there isn't a role there, only a playbook.

Reply:
I meant in the `aliases` file of the integration test, so that `setup_docker` is run before any of the tests for the `docker` connection plugin are executed.